notebooks/WFC3/calwf3_v1.0_cte/calwf3_with_v1.0_PCTE.ipynb

```diff
@@ -21,23 +21,25 @@
     "- Compare v1.0 and v2.0 products. \n",
```

Comment:
<p>The CI runs the execution twice, resulting in the error shown in the log here : <a href="https://github.com/spacetelescope/hst_notebooks/actions/runs/6883163300/job/18723158842?pr=100#step:8:621" target="_blank">https://github.com/spacetelescope/hst_notebooks/actions/runs/6883163300/job/18723158842?pr=100#step:8:621</a></p><p>The error occurs because the file `idv404axq_flt.fits' already exists. To prevent this, you can run calibration only if the file doesn't exist from the previous run. </p><p> </p><pre class="ql-syntax" spellcheck="false">if not os.path.exists('idv404axq_flt.fits'):
&nbsp; &nbsp; calwf3('idv404axq_raw.fits')
</pre><p> </p><p>Additionaly, there might also be a flag you can pass to this method in calwf3 to overwrite if file exists. It's something you can look into from the docs. Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>